server-extension/src/main/java/flyway/olcesium/V1_1__set_layers.java

```diff
@@ -0,0 +1,207 @@
+            layers.stream().forEach((layer) -> {
+                try {
+                    layerSelectionPluginLayers.put(layer.getId());
+                } catch (Exception ex) {
+                    LOG.warn("Updating Cesium views / could not list layers to background selection plugin");
+                }
+            });
+
+            for(View view : views) {
+                View modifyView = viewService.getViewWithConf(view.getId());
+                Bundle mapBundle = modifyView.getBundleByName(MAP_BUNDLE_NAME);
+                if (mapBundle != null) {
+
+                    JSONObject state = JSONHelper.createJSONObject(mapBundle.getState());
+                    if (state != null) {
+                        state.put("selectedLayers", selectedLayers);
+                        mapBundle.setState(state.toString());
+                        viewService.updateBundleSettingsForView(view.getId(), mapBundle);
+                    } else {
+                        LOG.warn("Updating Cesium views / mapfull state is null - view: ", view.getId());
+                    }
+
+                    JSONObject config = JSONHelper.createJSONObject(mapBundle.getConfig());
+                    if (config != null) {
+                        JSONArray plugins = config.optJSONArray("plugins");
+                        if (plugins != null) {
+                            for (int i = 0; i < plugins.length(); i++) {
+                                JSONObject plugin = plugins.getJSONObject(i);
+                                if (plugin.getString("id").contains(BASELAYER_PLUGIN_NAME)) {
+                                    JSONObject pluginConf = plugin.optJSONObject("config");
+                                    if (pluginConf == null) {
+                                        pluginConf = new JSONObject();
+                                        plugin.put("config", pluginConf);
+                                    }
+                                    pluginConf.put("baseLayers", layerSelectionPluginLayers);
+                                }
+                            }
+                        }
+                        mapBundle.setConfig(config.toString());
+                        viewService.updateBundleSettingsForView(view.getId(), mapBundle);
+                    } else {
+                        LOG.warn("Updating Cesium views / mapfull condig is null - view: ", view.getId());
+                    }
+
+                } else {
+                    LOG.warn("Updating Cesium views / mapfull bundle not found - view: ", view.getId());
+                }
+            }
+        } catch (Exception e) {
+            LOG.error(e, "Error setting selected layer for cesium views");
+        }
+    }
+
+    private List<View> getCesiumViews(Connection conn) throws SQLException {
+        List<View> list = new ArrayList<>();
+        final String sql = "SELECT id FROM portti_view WHERE name = '"+CESIUM_VIEW_NAME+"'";
```

Comment:
While there's no risk of SQL injection here, I'd still personally use `"... WHERE name = ?";`